integration-tests/endToEndFlow.spec.ts

```diff
@@ -0,0 +1,151 @@
+import axios from 'axios'
+import { parse } from 'node-html-parser'
+import * as CSV from 'csv-string'
+import {
+  AUDIT_BUCKET_NAME,
+  END_TO_END_TEST_DATE_PREFIX,
+  END_TO_END_TEST_EVENT_ID,
+  END_TO_END_TEST_FILE_NAME
+} from './constants/awsParameters'
+import {
+  endToEndTestRequestDataNoMatch,
+  endToEndTestRequestDataWithMatch
+} from './constants/requestData'
+import { retrieveSecureDownloadDbRecord } from './utils/aws/retrieveSecureDownloadDbRecord'
+import { copyAuditDataFromTestDataBucket } from './utils/aws/s3CopyAuditDataFromTestDataBucket'
+import { deleteAuditDataWithPrefix } from './utils/aws/s3DeleteAuditDataWithPrefix'
+import { getEnv, pause } from './utils/helpers'
+import { approveZendeskTicket } from './utils/zendesk/approveZendeskTicket'
+import { createZendeskTicket } from './utils/zendesk/createZendeskTicket'
+
+describe('Query results generated', () => {
+  jest.setTimeout(60000)
+
+  const getSecureDownloadPageHTML = async (
+    downloadHash: string
+  ): Promise<string> => {
+    try {
+      const response = await axios({
+        url: `${getEnv('QUERY_RESULTS_SECURE_DOWNLOAD_URL')}/${downloadHash}`,
+        method: 'POST'
+      })
+      return response.data
+    } catch (error) {
+      console.log(error)
```

Comment:
Probably better to use console.error to make it stand out

Reply:
Done.